learn/what_is_meilisearch/comparison_to_alternatives.mdx

```diff
@@ -106,12 +107,25 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 | Distinct attributes <br /><div style={{color: "#A9A9A9", fontSize: "0.9em"}}>De-duplicate documents by a field value</div>| ✅ | ✅ | ✅  | ✅ |
 | Grouping <br /><div style={{color: "#A9A9A9", fontSize: "0.9em"}}>Bucket documents by field values</div> | ❌ | ✅ | ✅  | ✅ |
 
+##### AI Search
+
+|   | Meilisearch | Algolia | Typesense | Elasticsearch |
+|---|:---:|:----:|:---:|:---:|
+| Semantic Search | ✅ | 🔶 <br /> Under Premium plan | ✅ | ✅ |
+| Hybrid Search | ✅ | 🔶 <br /> Under Premium plan | ✅ | ✅ |
+| Embedding Generation | ✅  <br /> OpenAI <br /> HuggingFace <br /> | Undisclosed | <br/> OpenAI <be /> Google PaLM <br /> GCP Vertex AI | ❌  |
+| Prompt Embedding | ✅ | Undisclosed | ❌ | ❌ | 
```

Comment:
Not sure about the naming here, but I don't have a better alternative for now. @mayasssh any suggestions?